print/components/config/Cover.vue

```diff
@@ -1,13 +1,13 @@
 <template>
   <div class="tw-break-after-page tw-text-center">
-    <div class="tw-font-medium tw-mt-20 tw-text-xl">{{ camp.name }}</div>
+    <p class="tw-font-medium tw-mt-20 tw-text-3xl">{{ camp.organizer }}</p>
 
-    <div :id="`content_${index}_cover`" class="tw-my-12 tw-text-6xl tw-font-semibold">
-      {{ camp.title }}
-    </div>
-    <div class="tw-text-3xl">
+    <h1 :id="`content_${index}_cover`" class="tw-my-16 tw-text-6xl tw-font-semibold">
+      {{ camp.title || camp.shortname }}
```

Comment:
In the issue we said that the shortname should always be displayed. If we have place, we also show the title